spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,6 +32,55 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
+[[microprofile5.0-RC1]]
```

Comment:
Per the following comments, I think it's best to have a look at the definition of the 4.1 release in this file. It has been updated a lot compared to 4.0.

Reply:
I think in addition to your recommendation about having a look at 4.1 definition, maybe this link could summarize the most important about  5.0: 

https://projects.eclipse.org/projects/technology.microprofile/releases/5.0

what do you think ?

Reply:
Correct, this is the link to the release plan that should contain all necessary details of the release